src/rtc.c

```diff
@@ -0,0 +1,792 @@
+/**
+ * @file rtc.c
+ * @brief Real-Time Clock Subsystem
+ * @ingroup rtc
+ */
+
+#include <string.h>
+#include <time.h>
+#include "libdragon.h"
+#include "system.h"
+
+/**
+ * @defgroup rtc Real-Time Clock Subsystem
+ * @ingroup libdragon
+ * @brief Real-time clock interface.
+ *
+ * The Joybus real-time clock is a cartridge peripheral that uses a battery
+ * to power a clock that tracks the date, time, and day of the week. The
+ * real-time clock keeps running even when the N64 is powered-off. The
+ * Joybus RTC is accessed through the serial interface (SI) similar to EEPROM
+ * and controllers. The Joybus RTC was only ever available on one official
+ * cartridge that was only available in Japan: Dōbutsu no Mori (Animal Forest).
+ * There is also a real-time clock included in the N64DD hardware, which uses
+ * a different interface and is not currently supported by libdragon.
```

Comment:
I think here we should at least mention that RTC is supported by 64drive and Everdrive (and emulators), so it can still be useful for homebrew development, otherwise it seems weird to an user why libdragon should care about a peripheral that was used once in history.

Reply:
Fixed in https://github.com/DragonMinded/libdragon/pull/152/commits/544baa50639e55196c93c17c97b8cda1365f7bfc